BVLinearGradient/BVLinearGradient.m

```diff
@@ -1,27 +1,39 @@
 #import "BVLinearGradient.h"
+
+
 #import <React/RCTConvert.h>
 #import <UIKit/UIKit.h>
 #import <QuartzCore/QuartzCore.h>
 
+#import "BVLinearGradientLayer.h"
+
 @implementation BVLinearGradient
 
 + (Class)layerClass
 {
-  return [CAGradientLayer class];
+  return [BVLinearGradientLayer class];
 }
 
 - (CAGradientLayer *)gradientLayer
 {
-  return (CAGradientLayer *)self.layer;
+  return (BVLinearGradientLayer *)self.layer;
```

Comment:
The method signature is `- (CAGradientLayer *)gradientLayer`, shouldn't it be:  `- (BVLinearGradientLayer *)gradientLayer`

Reply:
Yes you are right that's a copy-and-paste error